jvb/src/main/kotlin/org/jitsi/videobridge/dcsctp/DcSctpTransport.kt

```diff
@@ -40,19 +42,51 @@ class DcSctpTransport(
     parentLogger: Logger
 ) {
     val logger = createChildLogger(parentLogger)
-    lateinit var socket: DcSctpSocketInterface
+    private val lock = Any()
+    private var socket: DcSctpSocketInterface? = null
 
     fun start(callbacks: DcSctpSocketCallbacks, options: DcSctpOptions = DEFAULT_SOCKET_OPTIONS) {
-        socket = factory.create(name, callbacks, null, options)
+        synchronized(lock) {
+            socket = factory.create(name, callbacks, null, options)
+        }
     }
 
     fun handleIncomingSctp(packetInfo: PacketInfo) {
         val packet = packetInfo.packet
-        socket.receivePacket(packet.getBuffer(), packet.getOffset(), packet.getLength())
+        synchronized(lock) {
+            socket?.receivePacket(packet.getBuffer(), packet.getOffset(), packet.getLength())
```

Comment:
Do we care about keeping track of how often this happens?

Reply:
I don't think so - it's normal in the case where a sctp packet arrives after we shut down the sctp stack but before we shut down the dtls stack.  This is currently cluttering up the logs and is harmless, thus this change.